magenta/models/melody_rnn/melody_rnn_generate.py

```diff
@@ -217,6 +217,8 @@ def run_with_flags(generator):
 
 def main(unused_argv):
   """Saves bundle or runs generator based on flags."""
+  config = melody_rnn_config.config_from_flags()
+  config.hparams.temperature = FLAGS.temperature
```

Comment:
I think we should set the default for the flag to None and only override if it is given so as to avoid unintentionally overriding the default of the config.